ephios/plugins/questionnaires/templates/questionnaires/question_form.html

```diff
@@ -0,0 +1,92 @@
+{% extends "base.html" %}
+{% load bootstrap %}
+{% load crispy_forms_filters %}
+{% load i18n %}
+{% load formset_tags %}
+
+{% block title %}
+    {% if question %}
+        {% translate "Edit question" %}
+    {% else %}
+        {% translate "Create new question" %}
+    {% endif %}
+{% endblock %}
+
+{% block content %}
+    <div class="page-header">
+        {% if question %}
+            <h2>{% translate "Edit question" %}</h2>
+        {% else %}
+            <h2>{% translate "Create new question" %}</h2>
+        {% endif %}
+    </div>
+
+    <form method="post">
+        {% csrf_token %}
+        {{ form|crispy }}
+        <div id="choices-formset" class="formset" data-formset data-formset-prefix="{{ form.choices.prefix }}">
+            {{ form.choices.management_form }}
+            <div class="card mb-2">
+                <div class="card-body">
+                    <h5 class="card-title">
+                        {% translate "Choices" %}
+                    </h5>
+                    <p class="text-muted">{% translate "For single- and multiple choice questions" %}</p>
```

Comment:
please hide this section if it is not applicable

Reply:
Implemented in 04e070b4300e390c1a96fa6628272f72dde611f7